### ISSUE TITLE
kb0263_rename_pe_master tries to connect to old hostname and fails 

### ISSUE THREAD

**Issue by tkishel** [puppetlabs/support-tasks#47]
`kb0263_rename_pe_master` fails on at least PE 2019.1 ...

```
[root@pe-201910-master ~]# puppet task run support_tasks::kb0263_rename_pe_master -n pe-201910-master.puppetdebug.vlan
Starting job ...
Note: The task will run only on permitted nodes.
New job ID: 2
Nodes: 1

Started on pe-201910-master.puppetdebug.vlan ...
Couldn't retrieve job status: puppetlabs.orchestrator/unknown-error: org.apache.http.ConnectionClosedException: Connection closed
[root@pe-201910-master ~]# tail /var/log/messages 
May 24 06:37:20 pe-201910-master systemd: Stopping pe-nginx - Puppet Enterprise web server...
May 24 06:37:20 pe-201910-master systemd: Stopped pe-nginx - Puppet Enterprise web server.
May 24 06:37:21 pe-201910-master systemd: Stopping pe-orchestration-services Service...
May 24 06:37:51 pe-201910-master systemd: Stopped pe-orchestration-services Service.
May 24 06:37:52 pe-201910-master systemd: Stopping PCP Execution Protocol (PXP) Agent...
May 24 06:37:52 pe-201910-master systemd: Stopped PCP Execution Protocol (PXP) Agent.
May 24 06:38:00 pe-201910-master puppet-agent[17447]: csr_attributes file loading from /etc/puppetlabs/puppet/csr_attributes.yaml
May 24 06:38:00 pe-201910-master puppet-agent[17447]: Creating a new SSL certificate request for pe-201910-master-new.puppetdebug.vlan
May 24 06:38:00 pe-201910-master puppet-agent[17447]: Certificate Request fingerprint (SHA256): EC:6D:5C:F0:92:A5:DF:27:96:29:0D:64:0E:CD:5C:21:1D:8F:9F:31:C8:08:76:5F:85:6A:35:07:F7:8D:02:5C
May 24 06:38:00 pe-201910-master puppet-agent[17447]: Could not run: Failed to open TCP connection to pe-201910-master.puppetdebug.vlan:8140 (Connection refused - connect(2) for "pe-201910-master.puppetdebug.vlan" port 8140)
```

Internal-only ticket: https://tickets.puppetlabs.com/browse/SUP-1217

**Comment by MartyEwings** [comment]
https://github.com/puppetlabs/support-tasks/pull/53

**Comment by MartyEwings** [comment]
closed to to pr #53
